### ISSUE TITLE
List shared contents

### ISSUE THREAD

**Issue by alberfd** [photoprism/photoprism#674]
Is there anyway to list the shared contents?
if you share for example multiple albums is difficult to know what collections are share. You need to open each album and check manually if is shared or not.

maybe idea to new feature request?

**Comment by lastzero** [comment]
Shared albums have a share icon on their cover (upper left corner). Do you need a search filter for this?

**Comment by alberfd** [comment]
yes, it is true. I did not see it because when the background image is light it is very difficult to notice it. A css improvement may be help with this.

**Comment by graciousgrey** [comment]
This has been improved with one of [our lates releases](https://docs.photoprism.app/release-notes/#february-23-2025).
